fuse/request.go

```diff
@@ -270,6 +270,14 @@ func (r *request) serializeHeader(outPayloadSize int) {
 			dataLength = 0
 		}
 	}
+	// The InitOut structure has 24 bytes in fuse version <= 22.
```

Comment:
there is similar code in https://github.com/hanwen/go-fuse/blob/master/fuse/protocol-server.go#L70, can you investigate why it is not working?

also, please use gerrit (see CONTRIBUTING).